azure_cv.js

```diff
@@ -0,0 +1,30 @@
+//AZURE CV IMAGE ANALYSIS
+const cognitiveServices = require('cognitive-services');
+
+var computerVision = cognitiveServices.computerVision({
```

Comment:
Just want to make sure that the goal is to identify objects, not faces.

Reply:
Yes. objects and colors to start. Eventually faces too, but the idea is that the AI can help the analysts identify objects and act as another pair of eyes in the search.